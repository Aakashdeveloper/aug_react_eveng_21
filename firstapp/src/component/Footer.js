import React from 'react';

const Footer = (props) => {
    return(
        <div>
            <hr/>
            <center>
                <h3>&copy; Developer Funnel {props.month} {props.year} </h3>
            </center> 
        </div>
    )
}

export default Footer;