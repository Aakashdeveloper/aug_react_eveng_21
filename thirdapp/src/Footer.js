import React from 'react';

const Footer = () => {
    return(
        <div>
            <hr/>
            <center>
                <h3>&copy; DeveloperFunnel</h3>
            </center>
        </div>
    )
}

export default Footer;