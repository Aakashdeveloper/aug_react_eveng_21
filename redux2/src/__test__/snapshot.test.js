import React from 'react';
import Footer from '../component/Footer';
import {create} from 'react-test-renderer'

describe('Snapshot Testing',() => {
    test('testing footer',() => {
        let tree = create(<Footer/>)
        expect(tree.toJSON()).toMatchSnapshot()
    })
})