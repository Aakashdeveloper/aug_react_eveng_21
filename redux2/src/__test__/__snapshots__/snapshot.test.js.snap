// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Testing testing footer 1`] = `
<footer>
  <div>
    <hr />
    Â© Developer Funnels
  </div>
</footer>
`;
